src/hooks/use-report-fetcher.tsx

```diff
@@ -113,6 +118,9 @@ export const useReportFetcher = (
 
     const fetchReportLogs = useCallback(
         (reportId: string, severityList: string[], reportType: ReportType, messageFilter: string) => {
+            if (!studyUuid) {
+                return;
+            }
```

Comment:
seems like a fix, not something relative to typescript migration no ?
Move it in a separeted PR maybe ?

Reply:
not a big deal